packages/flutter/lib/src/cupertino/radio.dart

```diff
@@ -297,7 +294,7 @@ class _CupertinoRadioState<T> extends State<CupertinoRadio<T>> with TickerProvid
       checked: widget._selected,
       selected: accessibilitySelected,
       child: buildToggleable(
-        mouseCursor: effectiveMouseCursor,
+        mouseCursor: widget.mouseCursor ?? _defaultMouseCursor,
```

Comment:
Can you set this as the actual default value for CupertinoRadio.mosueCursor or is it more complicated than that?